lms/djangoapps/bookmarks/api.py

```diff
@@ -90,4 +92,24 @@ def delete_bookmark(user, usage_key):
         ObjectDoesNotExist: If a bookmark with the parameters does not exist.
     """
     bookmark = Bookmark.objects.get(user=user, usage_key=usage_key)
+    _track_event('edx.bookmark.removed', bookmark)
```

Comment:
Can you move this after the `delete()` so that if that fails, we do not emit any event?